app/utils/bestuursorgaan-meta-ttl.js

```diff
@@ -0,0 +1,14 @@
+export const getBestuursorgaanMetaTtl = (bestuursorgaan) => {
+  if (!bestuursorgaan) {
+    return;
+  }
+  const bestuursorgaanUri = bestuursorgaan.uri;
+
+  return `
+    @prefix mandaat: <http://data.vlaanderen.be/ns/mandaat#> .
+    @prefix ext: <http://mu.semte.ch/vocabularies/ext/> .
+    @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
```

Comment:
Not sure why you need mandaat and xsd as prefix if they are not used....

Reply:
will remove